torchio/data/image.py

```diff
@@ -97,6 +102,8 @@ def __init__(
             path: Union[TypePath, Sequence[TypePath], None] = None,
             type: str = None,
             tensor: Optional[TypeData] = None,
+            h5DS: Optional[TypeData] = None,
```

Comment:
`TypeData` means a NumPy array or a PyTorch tensor. Is this annotation correct?

Reply:
Oops sorry. Its not. I will modify accordingly.